site/content/docs/concepts/pipelines.en.md

```diff
@@ -2,6 +2,10 @@ Having an automated release process is one of the most important parts of softwa
 
 In this section, we'll talk about using Copilot to set up a CodePipeline that automatically builds your service code when you push to your GitHub, Bitbucket or AWS CodeCommit repository, deploys to your environments, and runs automated testing.
 
+!!! Attention
```

Comment:
Should we merge this change? Since we'll have copilot v1.11 out first can we remove this line?